src/components/OcModal.vue

```diff
@@ -11,7 +11,7 @@
       >
         <div class="oc-modal-title">
           <oc-icon v-if="icon" :name="icon" :variation="variation" />
-          <h2 v-text="title" />
+          <h2 id="oc-modal-label" v-text="title" />
```

Comment:
Why not `oc-modal-title` as id? 🤔

Reply:
Also fine for me, saw the a11y example use `-label` and went with it..

Reply:
I'd prefer `oc-modal-title` 😁

Reply:
`oc-modal-title` it is, then